code/unclassified/src/leap_year/Leap_Year.java

```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+ 
+public class Leap_Year {
+
+	int year1, year2;
+	Scanner sc;
+	// getting value of the years from user
+	void getVal() {
+		
+		sc = new Scanner(System.in);
+ 
+		System.out.println("Finding leap years between two years");
+		
+		System.out.println("\nEnter the first year : ");
+		year1 = Integer.parseInt(sc.nextLine());
+		
+		System.out.println("Enter the second  : ");
+		year2 = Integer.parseInt(sc.nextLine());
+	}
+	// Printing no of leap year between two years
+	void checkLeapYear() {

```

Comment:
How about a boolean return value instead? Let the other functions handle the console related things.